kroxylicious-integration-tests/pom.xml

```diff
@@ -72,23 +72,11 @@
             <groupId>io.kroxylicious</groupId>
             <artifactId>kroxylicious-kms-provider-aws-kms-test-support</artifactId>
             <scope>runtime</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.hamcrest</groupId>
-                    <artifactId>hamcrest-core</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
Has a merge gone wrong? I don't see anything in this PR to stop these exclusions being necessary?